test-utils/src/main/scala/com/mesosphere/utils/zookeeper/ZookeeperServerExtension.scala

```diff
@@ -0,0 +1,23 @@
+package com.mesosphere.utils.zookeeper
+import org.junit.jupiter.api.extension.{AfterAllCallback, BeforeAllCallback, ExtensionContext}
+
+/**
+  * Provides a JUnit 5 extension for booting a Zookeeper server for tests.
+  *
+  * @see [[com.mesosphere.utils.mesos.MesosClusterExtension]] for an example.
+  */
+class ZookeeperServerExtension extends BeforeAllCallback with AfterAllCallback {
```

Comment:
interesting; so idea here is that we'll use the same zookeeper instance throughout the tests? (isolate each test by prefix or something?)

Reply:
This will be per testsuite as it's already the case for the [Scalatest fixture](https://github.com/mesosphere/usi/blob/master/test-utils/src/main/scala/com/mesosphere/utils/zookeeper/ZookeeperServer.scala).